.tito/tito.props

```diff
@@ -0,0 +1,5 @@
+[buildconfig]
+builder = tito.builder.Builder
+tagger = tito.tagger.VersionTagger
+changelog_do_not_remove_cherrypick = 0
+changelog_format = %s (%ae)
```

Comment:
Unfortunately, this won't work without changing the 'VERSION.cmake'. Take a look on 5b926c to get a picture.